src/main/java/Tasks.java

```diff
@@ -0,0 +1,23 @@
+public class Tasks {
```

Comment:
Perhaps you could name this class as "Task" (singular) instead as it refers to one task